### ISSUE TITLE
Doesn't work using gradle 2.12

### ISSUE THREAD

**Issue by Centril** [prezi/gradle-haskell-plugin#14]
```                                                   
$ gradlew                                                                                                 
                                                                                                               
FAILURE: Build failed with an exception.                                                                       
                                                                                                               
* Where:                                                                                                       
Build file 'D:\programming\TDA283-compiler-construction\basic\build.gradle' line: 1                            
                                                                                                               
* What went wrong:                                                                                             
A problem occurred evaluating project ':basic'.                                                                
> Failed to apply plugin [id 'haskell']                                                                        
   > Could not create an instance of type com.prezi.haskell.gradle.model.DefaultHaskellSourceSet_Decorated.    
                                                                                                               
* Try:                                                                                                         
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. 
                                                                                                               
BUILD FAILED                                                                                                   
                                                                                                               
Total time: 3.346 secs                                                                                         
```

**Comment by vigoo** [comment]
That's true, sorry. We are still using 2.4 because of the incompatibility of this and some other custom plugins. It is on my todo list to upgrade it but did not have the time to do it yet.

**Comment by vigoo** [comment]
Fixed in version 0.5
